website/blog/2022-09-06/index.md

```diff
@@ -49,13 +50,15 @@ The theme for this week is Azure Functions. We'll talk about ...
 
 
 ## Azure Functions and Serverless explained.
+
 Azure Functions is a serverless compute service that enables you to run code on-demand without explicitly provisioning or managing the underlying infrastructure. 
 
 Developing code that will run in a serverless environment departs from the traditional ASP.NET WebAPI technique of creating controllers with actions. Instead, as the name implies, we make functions as small snippets of functionality. An Azure Function persists these code snippets and meta-information concerning when and how it should get executed. The function sleeps until it has been invoked by a trigger, which wakes it up, runs the code snippet and returns to sleep.
 
 This behaviour allows for a very attractive pricing model where you only pay for the execution time of an Azure Function. If you write code that never gets executed, it won't cost you anything! That means you only pay and Azure Function when it is actually used.
 
 ## Provision a new Functions App.
```

Comment:
I think before we even get to this part - we lay out some scenarios that Functions would solve. That way the code example can solve one of those scenarios. Like an HTTP trigger that writes to a storage queue, then a queue trigger that writes to a database - it simulates an async messaging service ... or something.